Airports/EGCC/SMR/Regions.txt

```diff
@@ -2423,22 +2434,19 @@ N053.19.57.808 W002.18.31.673
 N053.19.58.257 W002.18.30.723
 N053.19.58.333 W002.18.30.831
 N053.19.57.872 W002.18.31.778
-;Northern Stands Construction (AIRAC 1708)
-Manchester smrCCdisused N053.21.58.115 W002.17.25.556
-N053.22.03.782 W002.17.16.465
-N053.22.00.569 W002.17.10.896
-N053.22.01.302 W002.17.09.402
-N053.22.04.331 W002.17.14.699
-N053.22.10.045 W002.17.06.128
-N053.22.11.169 W002.17.08.063
-N053.22.05.654 W002.17.17.347
-N053.22.05.526 W002.17.17.133
-N053.22.04.864 W002.17.17.347
-N053.22.04.575 W002.17.17.244
-N053.22.02.471 W002.17.20.833
-N053.22.03.473 W002.17.22.723
-N053.22.01.332 W002.17.26.329
-N053.22.00.833 W002.17.25.456
-N053.21.59.205 W002.17.28.033
-N053.21.58.777 W002.17.27.695
-N053.21.59.110 W002.17.27.190
\ No newline at end of file
+;Northern Stands Construction (AIRAC 1809)
+Manchester smrCCdisused N053.22.02.471 W002.17.20.828
```

Comment:
Don't think the word Manchester is required here? There are several instances of it up above as well!

Reply:
It should have been REGIONNAME Manchester